### PR TITLE
Added AcceptLanguage global parameter in Azure

### PR DESCRIPTION
**Pull request by stankovski**

### CODE REVIEW

**Comment by stankovski** [comment]
@azuresdkci retest this please

File: AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/SwaggerBat/AzureReport.Cs/AutoRestReportServiceForAzure.cs [hunk 1]
```diff
@@ -43,6 +43,10 @@ public partial class AutoRestReportServiceForAzure : ServiceClient<AutoRestRepor
         public SubscriptionCloudCredentials Credentials { get; private set; }
 
         /// <summary>
+        /// </summary>
```
Comment (ogail) [inline AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/SwaggerBat/AzureReport.Cs/AutoRestReportServiceForAzure.cs]: summary?

File: AutoRest/Generators/Azure.Common/Azure.Common/AzureCodeGenerator.cs [hunk 2]
```diff
@@ -216,6 +218,29 @@ public static void AddAzureProperties(ServiceClient serviceClient)
                 apiVersion.IsRequired = false;
             }
 
+            var acceptLanguage = serviceClient.Properties
+                .FirstOrDefault(p => AcceptLanguage.Equals(p.SerializedName, StringComparison.OrdinalIgnoreCase));
+            if (acceptLanguage == null)
+            {
+                acceptLanguage = new Property
```
Comment (ogail) [inline AutoRest/Generators/Azure.Common/Azure.Common/AzureCodeGenerator.cs]: Nit: may worth adding doc so comments on the generated code is not empty

**Comment by ogail** [comment]
LGTM

**Pull request merged**